src/Psalm/Type/Atomic/TLiteralClassString.php

```diff
@@ -16,9 +16,9 @@ class TLiteralClassString extends TLiteralString
      * Whether or not this type can represent a child of the class named in $value
      * @var bool
      */
-    public $definite_class = false;
+    public $definite_class = true;
 
-    public function __construct(string $value, bool $definite_class = true)
+    public function __construct(string $value, bool $definite_class = false)
```

Comment:
I find different default in constructor signature and property definition a little confusing, even though it shouldn't matter in practice.